cypress/e2e/audio-player/audio-player.spec.ts

```diff
@@ -179,15 +178,15 @@ describe("Audio player", () => {
         cy.openUserSettings("Appearance")
             .get(".mx_ThemeChoicePanel")
             .within(() => {
-                cy.get("[data-testid='theme-choice-panel-selectors']").within(() => {
+                cy.findByTestId("theme-choice-panel-selectors").within(() => {
                     // Enable light theme
                     cy.get(".mx_ThemeSelector_light").click();
 
                     // Assert that the radio button for light theme was checked
                     cy.get(".mx_StyledRadioButton_checked input[value='light']").should("exist");
                 });
 
-                cy.get("[data-testid='theme-choice-panel-highcontrast']").within(() => {
+                cy.findByTestId("theme-choice-panel-highcontrast").within(() => {
                     // Click the checkbox
                     cy.get("label .mx_Checkbox_background").click();
```

Comment:
Thanks for spotting! Per https://github.com/matrix-org/matrix-react-sdk/pull/10537 this should be able to be written simply like below:

````
cy.openUserSettings("Appearance")
   .get(".mx_ThemeChoicePanel")
   .findByLabelText("Use high contrast")
   .click({ force: true }); // force click because the size of the checkbox is zero
````

Reply:
@alunturner it was addressed with https://github.com/matrix-org/matrix-react-sdk/pull/10537/commits/bfa8152538a3a0804ac14e231578d0cba73aa308.